scripts/xtensa-build-zephyr.py

```diff
@@ -173,34 +173,13 @@ def parse_args():
 						help="Path to a non-default rimage signing key.")
 	parser.add_argument("-o", "--overlay", type=pathlib.Path, required=False, action='append',
 						default=[], help="Paths to overlays")
-	parser.add_argument("-z", "--zephyr-ref", required=False,
-						help="Initial Zephyr git ref for the -c option."
-						" Can be a branch, tag, full SHA1 in a fork")
-	parser.add_argument("-u", "--url", required=False,
-						default="https://github.com/zephyrproject-rtos/zephyr/",
-						help="URL to clone Zephyr from")
-	mode_group = parser.add_mutually_exclusive_group()
-	mode_group.add_argument("-p", "--west_path", required=False, type=pathlib.Path,
-				help="""Points to existing Zephyr project directory. Incompatible with -c.
-Uses by default path used by -c mode: SOF_TOP/zephyrproject.
-If zephyr-project/modules/audio/sof is missing then a
-symbolic link pointing to SOF_TOP directory will automatically be
-created and west will recognize it as its new sof module.
-If zephyr-project/modules/audio/sof already exists and is a
-different copy than where this script is run from, then the
-behavior is undefined.
-This -p option is always required_if the real (not symbolic)
-sof/ and zephyr-project/ directories are not nested in one
-another.""",
-	)
-	mode_group.add_argument("-c", "--clone_mode", required=False, action="store_true",
-				help="""Using west, downloads inside this sof clone a new Zephyr
-project with the required git repos. Creates a
-sof/zephyrproject/modules/audio/sof symbolic link pointing
-back at this sof clone.
-Incompatible with -p. To stop after downloading Zephyr, do not
-pass any platform or cmake argument.""",
-	)
+	parser.add_argument("-p", "--pristine", required=False, action="store_true",
+						help="Perform pristine build removing build cache.")
+	parser.add_argument("-u", "--update", required=False, action="store_true",
```

Comment:
No, because init can be done also when we do not update. Check my reply to

> This must be part of if args.update.

It seems weird to call it "init_and_update". It implicitly suggest that when you do not use this flag you do not perform `west init` what is not true.